server/src-lib/Hasura/RQL/Types/Common.hs

```diff
@@ -41,6 +41,7 @@ module Hasura.RQL.Types.Common
        , InputWebhook(..)
        , ResolvedWebhook(..)
        , resolveWebhook
+       , NonNegativeInt(..)
```

Comment:
```suggestion
       , NonNegativeInt
```
Exposing the constructor of `NonNegativeInt` may not be a very good idea. Instead, you can expose the `mkNonNegativeInt` function that I have suggested.